docs/azure_monitor/export/export.trace.rst

```diff
@@ -0,0 +1,9 @@
+Span Exporter
+=============
+
+
+
+.. automodule:: azure_monitor.export.trace
```

Comment:
Should we really expose `AzureMonitorSpanExporter.export`? Seems like it's not part of the API as we do not expect users to call it directly.